assets/html/scss/documenter/_variables.scss

```diff
@@ -89,3 +89,15 @@ $documenter-docstring-header-padding: 0.5rem $documenter-container-left-padding;
 $documenter-docstring-body-padding-h: $documenter-container-left-padding;
 $documenter-docstring-body-padding-v: 0.75rem;
 $documenter-docstring-body-padding: $documenter-docstring-body-padding-v $documenter-docstring-body-padding-h;
+
+// Search Results variables
+$search-result-link-hover: rgba(0, 128, 128, 0.1) !default;
+
+$search-333: #333 !default;
+$search-f1f5f9: #f1f5f9 !default;
+$search-whitesmoke: whitesmoke !default;
```

Comment:
I think the variables should have a bit more "semantic" names. What role do they fill?